### ISSUE TITLE
Feature Request: disable sessions checks

### ISSUE THREAD

**Issue by ghost** [jonathanio/monitoring-nagios-haproxy#3]
In our setup we have just one connection per backend. So we always get an error for using more than 90% of the available sessions (1 of 1). I couldn't find a way to disable sessions check or tell them that 100% is still ok (101% seem to be changed to 90%).

Can you please add an options to disable session checks at all?

**Comment by jonathanio** [comment]
@novij,

Agreed. It is an edge case which had not been considered. [Line 457](https://github.com/jonathanio/monitoring-nagios-haproxy/blob/master/check_haproxy#L457) is your issue with `101`%, as it falls outside the allowed bounds and therefore uses the default value.

I'll see what I can do.

**Comment by frisi** [comment]
just for the record: the fork https://github.com/montaguethomas/monitoring-nagios-haproxy works around the same issue with the --nolimits option
